### DISCUSSION TITLE
Standardize interactions and independent form catalog number selection forms

### DISCUSSION THREAD

**Discussion by acbentley** [specify/specify7#5982] [Feature Requests]
It would be much clearer and more user-friendly for users if we could standardize the method of selecting catalog numbers to add to both transactions (like loans and gifts) and the new independent form selector developed by Jason

Currently, the transactions dialog looks like this:

![image](https://github.com/user-attachments/assets/59a2aff7-95c9-475e-8b6f-46cc18a98b86)

which includes options to use a record set or enter catalog numbers directly but does not include an option to use the query builder to search for records

Currently, the independent form option looks like this:

![Screenshot 2024-08-23 at 9 08 09 AM](https://github.com/user-attachments/assets/76386d94-ef4e-4417-85eb-d92c67f3dbbc)

which includes options to enter catalog numbers directly or use the query builder but does not include an option to use record sets

Could we standardize this dialog to provide a seamless experience for the user with a standard dialog that would include all three elements above - enter catalog numbers, use record sets, or use the query builder?

**Reported By**
Biodiversity Institute

**Additional context**

**Comment by grantfitzsimmons** [comment]
Thank you @acbentley !
